Subscriptions/routes/subscriptionsRouter.ts

```diff
@@ -5,16 +5,21 @@ import {
   getSubscriptionsForMember,
   getAllSubscriptions,
 } from "../BLL/subscriptionsBLL";
+import { SubscriptionObject } from "../interfaces/mongoose.gen";
+
 const router = Router();
 
 router.post("/", async (req, res, next) => {
   try {
     const movieId = req.body.movieId as ObjectId;
     const memberId = req.body.memberId as ObjectId;
     const date = req.body.date as Date;
-    await postSubscription(movieId, memberId, date);
-    res.status(201).json({ message: "ok" });
+    const postResponse: {
+      subscription: SubscriptionObject;
+    } = await postSubscription(movieId, memberId, date);
+    res.status(201).json({ message: "ok", ...postResponse });
```

Comment:
Please search a library that contains consts that represent all the HTTP STATUS CODE instead of using the direct number - it will be more clear. For example: https://github.com/prettymuchbryce/http-status-codes